nx

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+[ -f "local.env" ] || touch local.env
+# Make sure the docker network exists
+docker network create wbs-dev > /dev/null 2>&1 || true
```

Comment:
Move below dockerenv condition...

Reply:
Actually this needs to be created either way as the test-services also rely upon it already existing.